api/swagger-spec/swagger.yml

```diff
@@ -0,0 +1,20 @@
+swagger: "2.0"
```

Comment:
Are you using a swagger editor to create this file or you are writing the yaml files from scratch?